src/components/pin.tsx

```diff
@@ -35,11 +39,26 @@ export interface PinInterface {
 }
 
 export const Pin = forwardRef(
-  ({title, subtitle, onPin, onLock, additionButton}: PinProps, ref) => {
+  (
+    {
+      title,
+      subtitle,
+      onPin,
+      onLock,
+      additionButton,
+      i18n = undefined,
+      i18params = undefined,
+    }: PinProps,
+    ref,
+  ) => {
     const insets = useSafeAreaInsets();
     const [pin, setPin] = useState('');
     const [error, setError] = useState('');
     const [locked, setLocked] = useState<Date | null>(null);
+    const valueTitle = useMemo(
+      () => (typeof i18n !== 'undefined' ? getText(i18n, i18params) : title),
```

Comment:
You can directly pass `i18nParams` and `i18n` to the Text component

Reply:
I fix it

Reply:
![image](https://user-images.githubusercontent.com/69737893/206123139-16856ac8-0f7e-4d8d-a1d5-c2adcdd13982.png)